cedar-policy/src/api.rs

```diff
@@ -4356,3 +4356,26 @@ pub fn compute_entity_manifest(
 ) -> Result<EntityManifest, EntityManifestError> {
     entity_manifest::compute_entity_manifest(&schema.0, &pset.ast).map_err(|e| e.into())
 }
+
+/// Functions to make manipulating policies easier.
+pub mod policy_manipulation_functions {
+    use super::EntityUid;
+    use super::Policy;
+
+    /// Get all entity literals occuring in a `Policy`
+    pub fn get_entity_literals(p: Policy) -> Vec<EntityUid> {
```

Comment:
Rather than nesting this API inside of a module, I feel like it would make more sense to make it a method on policies?

Reply:
Mostly did this because I'd make it a free function in C++, but I can't find a consensus for Rust. Open to changing if other people agree.

Reply:
(Also note that we'll do #1120 soon, at which point organizing things into a module starts to make more sense)

Reply:
I don't have strong feelings - I'll defer to @cdisselkoen and/or @john-h-kastner-aws.

Reply:
`Policy::entity_literals()` makes a little more sense to me, but I don't feel super strongly.